src/migrations/Migration20230724204141.ts

```diff
@@ -0,0 +1,33 @@
+import { Migration } from '@mikro-orm/migrations';
+
+export class Migration20230724204141 extends Migration {
+  public async up(): Promise<void> {
+    if (process.env.CHAIN_ID == '1') {
+      this.addSql("UPDATE registry_operator SET module_address = '0x55032650b14df07b85bF18A3a3eC8E0Af2e028d5'");
```

Comment:
Is it a temporary thing? I think migrations shouldn't have any dependencies with ENV, app config and any other things.